docs/it/trust_marks_cie.rst

```diff
@@ -32,124 +55,31 @@ La tabella seguente riassume tutti i profili disponibili supportati per tutte le
       - **Descrizione**
       - **Tipi di entità sub**
     * - **public**
-      - l’entità nel claim *sub* appartiene alla pubblica amministrazione italiana
+      - l'entità nel claim *sub* appartiene alla pubblica amministrazione italiana
       - Tutte
     * - **private**
-      - l’entità nel claim *sub* appartiene al settore privato.
+      - l'entità nel claim *sub* appartiene al settore privato.
       - Tutte
     * - **web**
-      - l’entità nell attributo *sub* è compatibile con CIE-OIDC-CORE
+      - l'entità nell attributo *sub* è compatibile con CIE-OIDC-CORE
       - RP
     * - **native**
-      - l’entità nell attributo *sub* è compatibile con CIE-OIDC-MOBILE – non ancora supportato
+      - l'entità nell attributo *sub* è compatibile con CIE-OIDC-MOBILE – non ancora supportato
       - RP
     * - **underage**
-      - l’entità nel claim *sub* fornisce servizi online per underage (non ancora supportato)
+      - l'entità nel claim *sub* fornisce servizi online per underage (non ancora supportato)
       - RP
     * - **aggregator**
```

Comment:
Modificare in **intermediary** o, meglio ancora, allineare con SPID se previsto un profilo di questo tipo

Reply:
non vedo specifiche per spid minori oidc, quindi per non bloccare l'uscita su docs italia consiglio di rimuovere underage @sp640919

Reply:
rimosso

Reply:
Modificare in intermediary

Reply:
Non è previsto in SPID il profilo per i SA?

Reply:
Su questo fate proposte e guidate con PR/Revisioni, io vi supporto